nodes/DelAttr.py

```diff
@@ -1,3 +1,21 @@
+"""
+DelAttr astroid node
+
+DelAttr deletes the named attribute, provided the object allows it. For example,
+delattr(x, 'foobar') is equivalent to del x.foobar.
+
+Attributes:
+    - object  (object)
```

Comment:
I wonder if `object` here is really an instance of any Python class, or whether it's supposed to be a particular type of astroid Node.